includes/class-simple-local-avatars.php

```diff
@@ -306,6 +433,125 @@ public function admin_init() {
+
+		add_action( 'wpmu_options', array( $this, 'show_network_settings' ) );
+		add_action( 'update_wpmu_options', array( $this, 'save_network_settings' ) );
+	}
+
+	/**
+	 * Show the network settings
+	 */
+	public function show_network_settings() {
+		$mode = $this->get_network_mode();
+		?>
+
+        <h2><?php esc_html_e( 'Simple Local Avatars Settings', 'simple-local-avatars' ); ?></h2>
+        <table id="simple-local-avatars" class="form-table">
+            <tr>
+                <th scope="row">
+					<?php esc_html_e( 'Mode', 'simple-local-avatars' ); ?>
+                </th>
+                <td>
+                    <fieldset>
+                        <legend class="screen-reader-text"><?php esc_html_e( 'Mode', 'simple-local-avatars' ); ?></legend>
+                        <label><input name="simple_local_avatars[mode]" type="radio" id="sla-mode-default" value="default"<?php checked( $mode, 'default' ); ?> /> <?php esc_html_e( 'Default to the settings below when creating a new site', 'simple-local-avatars' ); ?></label><br/>
+                        <label><input name="simple_local_avatars[mode]" type="radio" id="sla-mode-enforce" value="enforce"<?php checked( $mode, 'enforce' ); ?> /> <?php esc_html_e( 'Enforce the settings below across all sites', 'simple-local-avatars' ); ?></label><br/>
+                    </fieldset>
+                </td>
+            </tr>
+            <tr>
+                <th scope="row">
+					<?php esc_html_e( 'Local avatars only', 'simple-local-avatars' ); ?>
+                </th>
+                <td>
+					<?php
+					$this->avatar_settings_field(
+						array(
+							'key'  => 'only',
+							'desc' => __( 'Only allow local avatars (still uses Gravatar for default avatars)	', 'simple-local-avatars' ),
+						)
+					);
+					?>
+                </td>
+            </tr>
+            <tr>
+                <th scope="row">
+					<?php esc_html_e( 'Local upload permissions', 'simple-local-avatars' ); ?>
+                </th>
+                <td>
+					<?php
+					$this->avatar_settings_field(
+						array(
+							'key'  => 'caps',
+							'desc' => __( 'Only allow users with file upload capabilities to upload local avatars (Authors and above)', 'simple-local-avatars' ),
+						)
+					);
+					?>
+                </td>
+            </tr>
+            <tr>
+                <th scope="row">
+					<?php esc_html_e( 'Shared network avatars', 'simple-local-avatars' ); ?>
+                </th>
+                <td>
+					<?php
+					$this->avatar_settings_field(
+						array(
+							'key'     => 'shared',
+							'desc'    => __( 'Uploaded avatars will be shared across the entire network, instead of being unique per site', 'simple-local-avatars' ),
+							'default' => 1,
+						)
+					);
+					?>
+                </td>
+            </tr>
+        </table>
+
+		<?php
+	}
+
+	/**
+	 * Handle saving the network settings
+	 */
+	public static function save_network_settings() {
```

Comment:
This function triggers some `WordPress.Security` warnings, not sure if we want to resolve every one or just add ignore rules to all of them.